### ISSUE TITLE
Google Chrome not working with "--use-angle=gl" tag

### ISSUE THREAD

**Issue by clippycoder** [eliemichel/MapsModelsImporter#13]
Hi. The instructions didn't seem to hard to follow, but I've run into the following issue: When I'm trying to inject into Chrome, if I use the  "--use-angle=gl" tag, then RenderDoc displays Open GLES active, but Chrome itself is a black screen. And if I remove the tag, then Chrome works fine, but RenderDoc either crashes or says Open GLES not presenting and is useless. In the video it seemed to work just fine. Any tips?

**Comment by eliemichel** [comment]
- What is your GPU?
 - What version of Chrome do you use? Avoid the last version, you can install for instance 70.0 using [this installer](https://www.slimjet.com/chrome/download-chrome.php?file=files/70.0.3538.77/ChromeStandaloneSetup64.exe) (and don't let it autoupdate)
 - What version of RenderDoc do you use?

**Comment by clippycoder** [comment]
Thanks for replying. My GPU is Intel HD Graphics 520. I've had the same result with multiple versions of chrome, both new and old, and I'm using the newest RenderDoc (v1.5).

**Comment by eliemichel** [comment]
The README specifies that RenderDoc v1.1 or v1.2 portable is needed. Anyway it is likely not working on your GPU, I fear. :/ Maybe try adding the `--ignore-gpu-blacklist` flag.

**Comment by clippycoder** [comment]
Hmm. I tried your suggestions, same result. I don't have another windows PC, but is there anyway to place those tags on a mac?

**Comment by eliemichel** [comment]
The add-on includes precompiled part that I built only for Windows (I don't have a Mac), and anyway I'm not sure that RenderDoc for Mac has the "inject into process" feature (linux does not for instance).
